### DISCUSSION TITLE
Outreachy Applicant

### DISCUSSION THREAD

**Discussion by AbeerTannous** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#47] [Outreachy 2022] [answered]
Hi I am Abeer Junior Full Stack Web Developer from the united states , I would like to contribute to this project please guide me through the process.

**Comment by jesutobi** [comment]
these are the names of the mentors @BatoolMM @alswajiab  , for every request you make tag them to it, i guess thats why this wasn't answered

**Comment by jesutobi** [comment]
but this link is all you would need to know about contributing to OSCSA https://www.outreachy.org/outreachy-december-2022-internship-round/communities/open-science-community-saudi-arabia-oscsa/

but to access this you would need to login to the outreachy portal...i hope this helps

**Comment by BatoolMM** [comment, marked as answer]
Thank you @jesutobi and Welcome, @AbeerTannous to the community!

We have two projects for Outreachy, you can find them here:
1- [Improve the bilinguality and accessibility of the website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/12)
2- Improve Localisation for [Open Educational Resources](https://github.com/Open-Science-Community-Saudi-Arabia/localisation/issues/13).

Some of the tasks in the issue have been addressed - you are also welcome to review open PR!

Let me know if you are still confused!

Feel free to comment under the issue, if you have any questions!
